proxy/src/project.rs

```diff
@@ -87,3 +86,47 @@ pub fn get(api: &coco::Api, project_urn: &coco::Urn) -> Result<Project, error::E
 
     Ok((project, stats).into())
 }
+
+/// Returns a list of `Project`s for your peer.
+pub fn list_projects(api: &coco::Api) -> Result<Vec<Project>, error::Error> {
+    let project_meta = api.list_projects()?;
+
+    project_meta
+        .into_iter()
+        .map(|project| {
+            api.with_browser(&project.urn(), |browser| {
+                let stats = browser.get_stats()?;
+                Ok((project, stats).into())
+            })
+        })
+        .collect()
+}
+
+/// Returns a stubbed feed of `Project`s
+pub fn discover() -> Result<Vec<Project>, error::Error> {
+    let urn = coco::Urn::new(
+        coco::Hash::hash(b"hash"),
+        coco::Protocol::Git,
+        coco::Path::parse("rad/issues/42")?,
+    );
+
+    let projects = vec![
```

Comment:
I would say add another project. Just to make sure that things look ok in the frontend when there's more than one datum.

Reply:
good call